packages/non-core/mongo-decimal/package.js

```diff
@@ -4,7 +4,7 @@ Package.describe({
 });
 
 Npm.depends({
-  "decimal.js": "10.2.1"
+  "decimal.js": "10.3.1"
```

Comment:
Hi @harryadel, sorry, but I didn't understand the purpose of bumping the version of this package without making any actual change to it. Could you explain it to me?